src/api/controllers/password.reset.js

```diff
@@ -59,16 +59,14 @@ export default class PasswordReset {
     const salt = genSaltSync(parseFloat(env.hashRounds));
     const hashedPassword = hashSync(userPassword, salt);
 
-    await User.update(
-      {
-        password: hashedPassword
-      },
-      {
-        where: {
-          email: userEmail
-        }
+    await User.update({
+      password: hashedPassword
```

Comment:
@Ramadhan0 this file also shouldn't be in your PR. Please, apply this feedback to all other files not directly related to your PR.